implementations/rust/ockam/ockam_command/src/vault/create.rs

```diff
@@ -1,16 +1,25 @@
 use crate::node::NodeOpts;
 use crate::util::{node_rpc, Rpc};
 use crate::CommandGlobalOpts;
+use crate::Result;
 use clap::Args;
 use ockam::Context;
+use ockam_api::config::cli;
 use ockam_api::nodes::models::vault::CreateVaultRequest;
 use ockam_core::api::Request;
+use ockam_vault::storage::FileStorage;
+use ockam_vault::Vault;
+use slug::slugify;
+use std::sync::Arc;
 
 /// Create vaults
 #[derive(Clone, Debug, Args)]
 pub struct CreateCommand {
     #[command(flatten)]
-    node_opts: NodeOpts,
+    node_opts: Option<NodeOpts>,
+
+    #[arg(short, long, conflicts_with = "node")]
+    vault_name: Option<String>,
```

Comment:
If you run the command as it is now, you will notice that if you provide no arguments the command wills still run because all arguments are optional. We want to require either `node_opts` or `vault_name`. The way to do that is:

The correct way to do this would be:
```rust
/// Create vaults
#[derive(Clone, Debug, Args)]
#[command(group(
    ArgGroup::new("n")
    .required(true)
    .args(["node", "vault_name"]),
))]
pub struct CreateCommand {
    #[command(flatten)]
    node_opts: Option<NodeOpts>,

    #[arg(long = "name")]
    vault_name: Option<String>,

    /// Path to the Vault storage file
    #[arg(short, long)]
    pub path: Option<String>,
}
```

Reply:
Also, you should run the commands on debug instead of release. Running the binary on release mode hides some clap errors.